docs/docusaurus/docs/cloud/connect/connect_airflow.md

```diff
@@ -1,7 +1,7 @@
 ---
-sidebar_label: 'Quickstart for GX Cloud and Airflow'
-title: 'Quickstart for GX Cloud and Airflow'
-id: airflow_quickstart
+sidebar_label: 'Connect GX Cloud to Airflow'
```

Comment:
This makes it sound like Airflow is a data source. A user does not connect GX to Airflow. Instead, they use Airflow to run their validations in a scheduled manner.

Reply:
@wookasz what verb would you prefer? It needs to be consistent with the other topics in the section. They all need to use the same verb.

Reply:
What are the verb options?

Reply:
@wookasz deploy, install, link, add.

There is a concern that _deploy_ might infer that the GX Agent is being installed in the user's production environment. It is for this reason that _connect_ was chosen as it's less likely to cause confusion. Keep in mind that a verb change will necessitate all of the topic titles in the section being updated and the TOC and landing pages being altered.

Reply:
From those, I would choose "integrate." A user is integrating GX data validation with their airflow pipeline.

Reply:
@wookasz, Unfortunately, _integrate_ doesn't work with the Python topic.  I recommended leaving the titles as is for now.

Reply:
I don't think one could defensibly argue that, but your call

Reply:
Agreed. I need to think about the best solution to address your concern.